lib/perl/Genome/Model/ReferenceSequence/Command/Converter/Create.pm

```diff
@@ -5,13 +5,14 @@ use warnings;
 
 use Genome;
 
+use Genome::Model::Build::ReferenceSequence::Converter;
 
 class Genome::Model::ReferenceSequence::Command::Converter::Create {
     is => 'Command::V2',
     has_input => [
         source_reference => { is => 'Genome::Model::Build::ReferenceSequence', doc => 'the reference to convert from' },
         destination_reference => { is => 'Genome::Model::Build::ReferenceSequence', doc => 'the reference to convert to' },
-        algorithm => { is => 'Text', doc => 'method to do the conversion (valid values are methods of Genome::FeatureList::Converter)' },
+        algorithm => { is => 'Text', doc => 'method to do the conversion', valid_values => $Genome::Model::Build::ReferenceSequence::Converter::VALID_ALGORITHMS, },
```

Comment:
This could refer to `Genome::Model::Build::ReferenceSequence::Converter->__meta__->property(name => 'algorithm')->valid_values` rather than requiring a separate `our` variable.